onnxruntime_extensions/__init__.py

```diff
@@ -10,6 +10,10 @@
 
 __author__ = "Microsoft"
 
+import importlib.util
+spec = importlib.util.find_spec('pybind11')
```

Comment:
why is pybind11 needed?

Reply:
pybind11 is required by _extensions_pydll. It will throw an exception: ModuleNotFoundError: No module named 'onnxruntime_extensions._extensions_pydll when importing onnxruntime_extensions when pybind11 is not installed

Reply:
removed the change for pybind11 as we discussed